ui/public/i18n/language.json

```
@@ -1202,11 +1202,11 @@
     "AllowGroupsToSSH_SFTP":"Select one or more groups to grant SSH and/or SFTP access. Administrators are always granted SSH and SFTP access. The root account is still subject to the \"Allow root login\" checkbox setting.",
     "ssh_everyone_info": "Set the basic permission for non administrative accounts.",
     "ssh_administrators_info": "Administrators are always granted SSH and SFTP access.",
-    "shell_bash_needed": "The shell Bash is needed to use the cockpit server manager and the SSH service, you can override the shell of all users if this option is enabled",
+    "shell_bash_needed": "The shell Bash is needed to use the Cockpit Server Manager and the SSH service, you can override the shell of all users if this option is enabled. Once enabled, the Shell setting returned by the account provider is ignored. As a consequence, non-admin users must be granted again SSH and SFTP access permissions.",
```

I think describing what happens when enabled is enough. I'd avoid the "you" form, too. The Bash requirement to access Cockpit is an internal detail we can avoid to communicate.
 
```suggestion
    "shell_bash_needed": "Once enabled, the Shell setting returned by the account provider is ignored. As a consequence, non-admin users must be granted again SSH and SFTP access permissions.",
```

What do you think?

---

I agree :+1: